asciidoc/modules/con_mg_changes-in-micrometer-metrics.adoc

```diff
@@ -0,0 +1,76 @@
+[id="changes-in-micrometer-metrics_{context}"]
+= Changes in Micrometer metrics
+
+The following section describes the changes in Micrometer metrics.
+
+== TCP sent and received bytes are recorded as counters with equivalent HTTP request and response summaries
+
+In prior releases, the following metrics were recorded as distribution summaries for sockets. From {VertX} {4} onward, these metrics are logged as counter, which report the amount of data exchanged.
+
+* Net client
++
+** `vertx_net_client_bytesReceived`
+** `vertx_net_client_bytesSent`
+
+* Net server
++
+** `vertx_net_server_bytesReceived`
+** `vertx_net_server_bytesSent`
+
+For these counters, equivalent distribution summaries have been introduced for HTTP. These summaries are used to account for request and response sizes.
+
```

Comment:
@vietj  Can I rephrase this sentence to:
For these counters, equivalent distribution summaries have been introduced for HTTP. These summaries are used to collect information about the request and response sizes.

Reply:
sure you can